service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go

```diff
@@ -95,15 +99,16 @@ func toSecret(ctx context.Context, v interface{}, config Config) (*corev1.Secret
 		},
 	}
 
-	marshalledRemoteWrites, err := yaml.Marshal(remoteWrites)
+	values := remoteWriteValues{RemoteWrite: remoteWrites}
```

Comment:
I nested it under `remoteWrite`

cause: https://github.com/giantswarm/prometheus-agent-app/blob/master/helm/prometheus-agent/values.yaml